mmdet3d/models/task_modules/coders/centerpoint_bbox_coders.py

```diff
@@ -36,7 +36,7 @@ def __init__(self,
         self.pc_range = pc_range
         self.out_size_factor = out_size_factor
         self.voxel_size = voxel_size
-        self.post_center_range = post_center_range
+        self.post_center_range = Tensor(post_center_range)
```

Comment:
Should notice that the default `post_center_range` is `None`, which may cause an error here.

Reply:
True, would it be viable to throw the NotImplementedError already upon initialization or why is there a default value at all if it is currently required?

Reply:
True, I updated it so that it stays None in that case.